dv360/src/client.ts

```diff
@@ -125,23 +108,17 @@ export class Client implements ClientInterface {
 
   readonly args: Required<ClientArgs>;
   readonly ruleStore: {
-    [ruleName: string]: RuleExecutor<
-      ClientInterface,
-      RuleGranularity,
-      ClientArgs,
-      Record<string, ParamDefinition>
-    >;
+    [ruleName: string]: RuleExecutor<DisplayVideoClientTypes>;
   };
 
   addRule<Params extends Record<keyof Params, ParamDefinition>>(
-    rule: RuleExecutorClass<
-      ClientInterface,
-      RuleGranularity,
-      ClientArgs,
-      Params
-    >,
+    rule: RuleExecutorClass<DisplayVideoClientTypes>,
     settingsArray: ReadonlyArray<string[]>,
   ): ClientInterface {
+    const ruule = this.ruleStore[rule.definition.name] as RuleExecutor<
```

Comment:
There is a typo here that will break the code: ruule should be rule

Reply:
Thanks for catching - it was an unused variable. It wouldn't have broken the code but definitely shouldn't be there.